lrpar/src/lib/ctbuilder.rs

```diff
@@ -865,7 +865,7 @@ where
             })
             .collect::<Vec<_>>();
         let rule_map_len = rule_map.len();
-        quote! {
+        let cache_info = quote! {
             #[allow(unused)]
             mod _cache_information_ {
```

Comment:
We may want to do something else here, as we no longer actually compile this code
as we change it it could perhaps start to contain syntax errors, etc.

We could emit *both* the cache_info module, and the module text within that constant string.
Or we could just emit a formatted string more like the original comment.

Reply:
Don't we just want to output the information in one place? Perhaps I've misunderstood!

Reply:
Yeah, I guess i'm saying:

this emits a strong:
```
const CACHE_INFO: &str = "#[allow(unused)] mod _cache_information_ {...
```

But because we never actually compile `mod _cache_information_` anymore, it is just embedded in that string,
so it is both confusing, and we could make a change that would cause `mod _cache_information_` to not actually compile but not notice.

I basically just did the minimal change to check that embedding it as a string works as a proof of concept.

Reply:
Should be fixed in [a687505](https://github.com/softdevteam/grmtools/pull/525/commits/a6875058c2a420a9c6848e6a35ab5c57d9dfd8d7)